tests/test_examples.py

```diff
@@ -0,0 +1,30 @@
+from importlib import import_module
+import unittest
+
+
+class TestExamples(unittest.TestCase):
+
+    def _run(self, name):
+        import_module('examples.'+name)
```

Comment:
This class will allow us to run the examples as part of the test suite.  I have seen these particular examples run in travis.yml